physicsnemo/registry/model_registry.py

```diff
@@ -16,6 +16,7 @@
 
 import warnings
 from importlib.metadata import EntryPoint, entry_points
+from importlib_metadata import EntryPoint as EntryPointOld
```

Comment:
Can you add a small comment explaining what this is, why it here?
We will forget in a month.

importlib_metadata is a back port of new APIs for previous python vesions, also users dont necessarily need this installed.  If it happens to be installed, we should check it.

Or does this come with base python (I dont think so but maybe)

Reply:
Best guess is somehow older versions of importlib are getting being used and some how this impacts the return type on `entry_points`

Reply:
What about this solution?

https://github.com/sphinx-doc/sphinx/issues/10157#issuecomment-1028068202